src/test/java/com/werfen/report/GridReportTest.java

```diff
@@ -61,25 +64,27 @@ private GridReportConfiguration getConfiguration(String fileName) throws IOExcep
                 .headerConfiguration(this.buildHeaderConfiguration())
                 .footerConfiguration(this.buildReportFooterConfiguration())
                 .gridColumnConfiguration(GridColumnConfiguration.builder().name("col1").width(GridReportColumnWidth.COLUMN_WIDTH_3).translation("col2").build())
+                .gridColumnConfiguration(GridColumnConfiguration.builder().name("col2").width(GridReportColumnWidth.COLUMN_WIDTH_3).translation("col3").build())
+                .gridColumnConfiguration(GridColumnConfiguration.builder().name("col3").width(GridReportColumnWidth.COLUMN_WIDTH_3).translation("col4").build())
                 .build();
     }
 
     private ReportHeaderConfiguration buildHeaderConfiguration() {
         return ReportHeaderConfiguration.builder()
                 .title("Grid report")
                 .logoPath("src/main/resources/AF_WERFEN_BLUE_POS_RGB.png")
-                .field1(ReportField.builder().name("Lab name").value("Name").build())
-                .field2(ReportField.builder().name("Second").value("Another").build())
-                .field3(ReportField.builder().name("Third").value("Another one").build())
-                .field4(ReportField.builder().name("Fourth").value("Last one").build())
+                .field1(GridReportField.of("Lab name","Name"))
```

Comment:
This update of tests is great, but I miss a case where the default value provided by the library (through GeneralConfiguration.getDefaultNullString()) is changed. Thus, I would try to include it. Probably that would mean creating a new golden PDF for grid reports, but I think it is worth covering a wider amount of scenarios.

Reply:
Create a test changing the configuration default value